control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -166,7 +166,10 @@ func (r *HostedControlPlaneReconciler) Reconcile(ctx context.Context, req ctrl.R
 			return ctrl.Result{}, fmt.Errorf("failed to add finalizer to hostedControlPlane: %w", err)
 		}
 	}
-
+	if cpoutil.IsReconciliationPaused(hostedControlPlane.Annotations) {
```

Comment:
Is there scenarios where we'd want pause but kee[ status to keep operating?

Reply:
I think if reconciliation is paused the whole process should be paused. But if others feel different we could move.